sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -90,6 +90,7 @@ export interface TracingContext {
 
 // @public
 export interface TracingSpan {
+    addEvent(name: string, attributesOrStartTime?: unknown, startTime?: unknown): void;
```

Comment:
https://github.com/Azure/azure-sdk-for-js/pull/31162/files will add support for this which you can then use (once merged) - you will need to use ^1.1.3 once it is published